ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/JsonRpcMethod.java

```diff
@@ -31,4 +34,16 @@
    * @return output from applying the JSON-RPC method to the input.
    */
   JsonRpcResponse response(JsonRpcRequest request);
+
+  /**
+   * The list of Permissions that correspond to this JSON-RPC method. e.g. [net/*, net/listening]
+   *
+   * @return list of permissions that match this method.
+   */
+  default List<String> getPermissions() {
```

Comment:
Have we considered supporting `*/*` as a valid permission config?

Reply:
kinda, we need to have a conversation about where and how we inflate the permissions list

Reply:
👍